docs/guide/images.md

```diff
@@ -356,4 +356,28 @@ ezpublish:
             provider: remote
             options:
                 url_pattern: 'http://example.com/var/site/storage/%%id%%'
-```
\ No newline at end of file
+```
+
+## Resizing images
+
+You can resize all images of a chosen Content Type using the `ezplatform:images:resize-original` command.
+You need to provide the command with:
+
+- identifier of the image Content Type
+- identifier of the Field you want to affect
+- name of the image variation to apply to the images
```

Comment:
LiipImagine uses name `filter` instead of `variation`. It's up to you to consider which one we prefer.

Reply:
The terminology is a bit all over the place, so we decided for `variation` for the whole page.